cloudinary-core/src/main/java/com/cloudinary/Url.java

```diff
@@ -371,7 +370,7 @@ public String generate(String source) {
         source = finalizedSource[0];
         String sourceToSign = finalizedSource[1];
 
-        if (forceVersion && sourceToSign.contains("/") && !StringUtils.hasVersionString(sourceToSign) &&
+        if (this.config.forceVersion && sourceToSign.contains("/") && !StringUtils.hasVersionString(sourceToSign) &&
```

Comment:
Small comment: Let's put the check if the user sent a version at the beginning.
If he did, no need to check the rest.